presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -632,6 +633,36 @@ public void rollback()
         // TODO: cleanup open transaction
     }
 
+    @Override
+    public void dropBranch(ConnectorSession session, ConnectorTableHandle tableHandle, Optional<String> branchName)
+    {
+        String branch = branchName.get().replace("'", "");
```

Comment:
Instead of `get()` maybe do `getOrElseThrow` and include a helpful error message?